tests/integration/adapter/test_basic.py

```diff
@@ -0,0 +1,25 @@
+from dbt.tests.adapter.basic.test_base import BaseSimpleMaterializations
+from dbt.tests.adapter.basic.test_empty import BaseEmpty
+from dbt.tests.adapter.basic.test_incremental import BaseIncremental
+from dbt.tests.adapter.basic.test_snapshot_timestamp import BaseSnapshotTimestamp
+from dbt.tests.adapter.basic.test_snapshot_check_cols import BaseSnapshotCheckCols
+
+
+class TestBaseSimpleMaterializations(BaseSimpleMaterializations):
+    pass
+
+
+class TestEmptyClickHouseAdapter(BaseEmpty):
+    pass
+
+
+class TestIncrementalClickHouseAdapter(BaseIncremental):
+    pass
+
+
+class TestSnapshotTimestampMyAdapter(BaseSnapshotTimestamp):
```

Comment:
Maybe remove the prefixes `ClickHouseAdapter` / `MyAdapter` and just use simple names `BaseSnapshotTimestamp` -> `TestSnapshotTimestamp`, `BaseEmpty` -> `TestEmpty`.

Reply:
Good idea - done